release_test_compare.py

```diff
@@ -330,8 +334,20 @@ def compare_coverage(sample, diffs, scenarios_reported, scenarios):
                 sample.name.startswith('90308A') and
                 MICALL_VERSION == '7.11'):
```

Comment:
You can get rid of special cases for old releases.

Reply:
Done!